### DISCUSSION TITLE
Lost access to my original github account because i acidentally ran an info stealer.

### DISCUSSION THREAD

**Discussion by lostaccesstomygithub** [community/community#127217] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As the title says, ran an info stealer, didn't think much (change my main emails and game accounts, didn't think someone would go for some random github account with much influence) but here we are, github gone, linked email in their hands, is there any way i can open a support ticket? i don't have much proof of ownership atm (considering everything i had is, poof) but i can try my best.

**Comment by lostaccesstomygithub** [comment]
wrong topic area, didnt see that button

**Comment by ebndev** [comment]
Hi @lostaccesstomygithub ,

You can open a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
